cli/internal/run/dry_run.go

```diff
@@ -42,6 +56,11 @@ func DryRun(
 	dryRunJSON := rs.Opts.runOpts.dryRunJSON
 	singlePackage := rs.Opts.runOpts.singlePackage
 
+	summary := &dryRunSummary{
+		Packages: packagesInScope,
+		Tasks:    []hashedTask{},
```

Comment:
Nit, we can avoid allocating this map: https://github.com/uber-go/guide/blob/master/style.md#omit-zero-value-fields-in-structs